package.json

```diff
@@ -1,22 +1,21 @@
 {
         "name": "zsock",
         "description": "A small library for opening Unix Domain Sockets in Solaris Zones.",
-        "version": "1.3.1",
+        "version": "2.0.0",
```

Comment:
Do we usually bump major versions when switching node versions?

Reply:
Because we're going from supporting node 0.8 to supporting only 0.10, that's a breaking change so I figured it was appropriate.